@ndlib/gatsby-theme-marble/src/components/Shared/SearchBox/SearchField/index.js

```diff
@@ -6,7 +6,7 @@ import { updateInput } from 'store/actions/searchActions'
 // You should only have one search field on a page.
 export const SearchField = ({ submitSearch, searchReducer, searchPath, dispatch, className, location }) => {
   const { rawInput } = searchReducer
-  const fieldLabel = 'Search the Collections'
+  const fieldLabel = 'Search our digitized artwork, rare books, artifacts, and archival materials'
```

Comment:
I feel like this line should be customizable in the configuration. 
Add a variable to site.siteMetadata in `/site/content/configuration.js` along the lines of `searchBoxDefaultText` set to this string. Then in the main `gatsby-node.js file` find this chunk:
```
  type SiteMetadata {
    universalViewerBaseURL: String
    googleMapApiURL: String
    iiifHelpURL: String
    searchBase: searchBase
    footerText: String
    menus: [menus]
    hideLogo: Boolean
    useLogin: Boolean
    userContentPath: String
    authClient: authClient
    searchPath: String
    languages: languages
  }
```
and add the variable there.

Next add a `useStaticQuery` to pull the variable into this component. Do a null check and default to "Search" if there is no string. Update tests for both cases when a string is found and when it is not.